xinference/core/worker.py

```diff
@@ -165,10 +166,15 @@ async def launch_builtin_model(
         llm_family, llm_spec, quantization = match_result
         assert quantization is not None
 
-        from ..model.llm.llm_family import cache
+        from ..model.llm.llm_family import cache, cache_peft
 
         save_path = await asyncio.to_thread(cache, llm_family, llm_spec, quantization)
 
+        if peft_model_path is not None:
```

Comment:
Since peft models can only be applied on pytorch models, how about raising an error when the format does't match?